src/hooks/useReimbursementAccountStepFormSubmit.ts

```diff
@@ -17,16 +18,18 @@ type UseReimbursementAccountStepFormSubmitParams = Pick<SubStepProps, 'isEditing
  * @param isEditing - if form is in editing mode
  * @param onNext - callback
  * @param fieldIds - field IDs for particular step
+ * @param shouldSaveDraft - if we should save draft values
  */
 export default function useReimbursementAccountStepFormSubmit({
     formId = ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT_FORM,
     isEditing,
     onNext,
     fieldIds,
+    shouldSaveDraft = false,
```

Comment:
I'm not convinced by this change. `isEditing` and `shouldSaveDraft` serve the exact same purpose now.

`isEdititing`: is in editing mode? -> should save draft.

`shouldSaveDraft`: should save draft.

What I suggested before was removing `isEditing`, and *pass* the local `isEditing` to `shouldSaveDraft` argument, like...

```ts
useReimbursementAccountStepFormSubmit({
    // ...
    shouldSaveDraft: isEditing,
    // ...
});
```

...in a few files using this util.

The PR diff will be slightly bigger, but this is not code golf.

Reply:
If you disagree with this, please provide some arguments.

Reply:
The code left as you suggested it leaves absolutely no trace of explanation _why_ we're doing what we're doing. How is anyone supposed to figure out tomorrow that...

_"phone number? save the draft now!"_

...means...

_"We want to trim the phone number so it doesn't have extra whitespace later"_

(with an extra context like "we're too afraid to do this on every screen for every input")

We started with...

_"phone number? lat's lie that we're in the editing mode!"_

So, still, we're going in a better direction, I think.